app/controllers/registrations_controller.rb

```diff
@@ -80,7 +80,7 @@ def build_devise_resource
     build_resource(sign_up_params)
     resource.saw_onboarding = false
     resource.registered_at = Time.current
-    resource.editor_version = "v2"
+    resource.setting&.editor_version = "v2"
```

Comment:
In this case, `resource` is the user, which is why I call `setting`. The `&` is for incase the `setting` record does not exist at this point; the `editor_version` defaults to 'v2' regardless